infrastructure/cli/environment.go

```diff
@@ -46,11 +46,12 @@ func AppendCliEnvironmentVariables(currentEnv []string, appendToken bool) (updat
 
 	// remove any existing env vars that we are going to set
 	valuesToRemove := map[string]bool{
-		ApiEnvVar:                                true,
-		TokenEnvVar:                              true,
-		DisableAnalyticsEnvVar:                   true,
-		auth.CONFIG_KEY_OAUTH_TOKEN:              true,
-		configuration.FF_OAUTH_AUTH_FLOW_ENABLED: true,
+		ApiEnvVar:   true,
+		TokenEnvVar: true,
+		configuration.AUTHENTICATION_BEARER_TOKEN: true,
```

Comment:
Issue: this constant is the config key, not the environment variable name. The reason is that there are multiple env vars as aliases, see [here](https://github.com/snyk/go-application-framework/blob/main/pkg/app/app.go#L110)